### PR TITLE
BM-273 bewotec https communication

### PR DESCRIPTION
**Pull request by EyesOnlyNet**

### CODE REVIEW

File: src/crsAdapter/BewotecExpertAdapter.js [hunk 1]
```diff
@@ -139,6 +283,284 @@ class BewotecExpertAdapter {
...
+            case SERVICE_TYPES.car: {
+                let serviceCode = crsService.servicecode;
+
+                // gaps in the regEx result array will result in lined up "." after the join
+                return !serviceCode || serviceCode.match(CONFIG.services.car.serviceCodeRegEx).join('.').indexOf('..') !== -1;
+            }
+            case SERVICE_TYPES.hotel: {
+                return !crsService.servicecode || !crsService.accomodation;
+            }
+            case SERVICE_TYPES.camper: {
```
Comment (tarleb) [inline src/crsAdapter/BewotecExpertAdapter.js]: Why not fall-through to `car` instead?

Comment (EyesOnlyNet) [reply src/crsAdapter/BewotecExpertAdapter.js]: you are completely right!

File: README.md [hunk 2]
```diff
@@ -98,32 +98,38 @@ useTimeFormat | 'HHmm' (according to [momentjs date format](https://momentjs.com
...
+|                 |                                   | environment                | \<environment\>
+|                 |                                   | exportId                   | '0123-456789-abcdef'
 
 (*) optional
 
 For some connections you need credentials or other connection data,
 which you can set in the `connectionOptions`.
 
-**[TOMA 2]** _**connectionUrl** is needed, when the adapter is not directly used in the first child window of the TOMA application_
+**[TOMA 2]** _connectionUrl_ is needed, when the adapter is not directly used in the first child window of the TOMA application_
```
Comment (tarleb) [inline README.md]: Looks like this line is missing a `_` somewhere (or the last one should be a period instead)

Comment (EyesOnlyNet) [reply README.md]: hm, you are right. it's lost somewhere.

**Pull request merged**